nixos/lib/testing/nodes.nix

```diff
@@ -78,14 +78,6 @@ in
       '';
     };
 
-    minimal = mkOption {
-      type = types.bool;
-      default = false;
-      description = mdDoc ''
-        Enable to configure all [{option}`nodes`](#test-opt-nodes) to run with a minimal kernel.
-      '';
-    };
-
```

Comment:
Could you add a `mkRemovedOptionModule` for this in `nixos/lib/testing/legacy.nix`?

Reply:
Sure, although using it already causes an error because `cifs_timeout_2_6_38` was removed aeons ago.

Reply:
Also,

https://github.com/NixOS/nixpkgs/blob/5d6fa6fdeb97ce5763f8bc2fa7b4185c3562c43a/nixos/lib/testing/legacy.nix#L6-L9

Should I just add it inside the comment?

Reply:
You can leave the comment as is and add a real `imports`, because in a comment it is useless.
In this case I don't care about making it a warning before turning it into an error, because
 - The functionality is already broken
 - I expect usage of `minimal` to be rare
 
So the comment only applies to the `nodes` migration it suggests for now.

Reply:
Ah, I didn't realise it was about warnings, not errors. Done.

Reply:
Ah, it does need `assertions`:

```
error: The option `assertions' does not exist.
```

Reply:
I wrote https://github.com/NixOS/nixpkgs/pull/207187 to resolve this tech debt, but don't block this PR for it.